azure-pipelines.yml

```diff
@@ -680,9 +680,6 @@ jobs:
 
 - job: BuildYumPackage
   displayName: Build Yum Package
-
-  dependsOn: BuildPythonWheel
-  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
```

Comment:
We should **build RPM/Yum for PRs**, like we **build MSI for PRs**. Otherwise, issue like this can't be discovered by PR, but only after the PR gets merged.